auth_u2f/models/u2f_device.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2017 Joren Van Onder
+# Copyright (C) 2019 initOS GmbH
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301 USA
+
+from u2flib_server.u2f import complete_registration
+
+from odoo import api, fields, models
+from odoo.http import request
+
+
+class U2FDevice(models.Model):
+    _name = 'u2f.device'
+
+    name = fields.Char(required=True)
+    json = fields.Char(
+        string='Scan',
+        required=True,
+        help='Technical data returned by u2flib or the browser'
+    )
+    user_id = fields.Many2one('res.users', required=True, readonly=True)
+    default = fields.Boolean(help='Device used during login.', readonly=True)
+
+    @api.model
+    def create(self, vals):
+        vals['user_id'] = self.env.user.id
```

Comment:
I'm not sure why this was added but I wouldn't do this. It prevents an administrator from registering U2F devices for employees which I think could be a common use case. Without this line the admin can go to the user form view and add a device in the U2F tab. With this line the device belongs to the admin instead.